evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -131,7 +130,7 @@ def test_make_pipeline(
                 if estimator_class.model_family == ModelFamily.LINEAR_MODEL
                 else []
             )
-            drop_null = [DropNullColumns] if "all_null" in column_names else []
+            drop_null = [DropColumns] if "all_null" in column_names else []
```

Comment:
This change is because all_null will be considered an unknown column now, rather than double right?

Reply:
Yes!